### DISCUSSION TITLE
Is Wren Dead?

### DISCUSSION THREAD

**Discussion by olivia-banks** [wren-lang/wren#1167] [General]
Sorry if this is a rude question, but is Wren dead? The code seems to have not been updated in a couple of years, and while there are issues and pull requests, no one seems to be able to merge them (the last merged PR was on September 28 of last year). I'd like to integrate Wren into my application, but I don't want to if it's no longer getting updated.

There seems to be nothing about this, and the website claims "active development".

**Comment by ruby0x1** [comment]
Nope! Quite far from it. I'll expand a bit more when I can but we work with and on Wren constantly in [luxe](https://createwith.luxe) and busy shipping a commercial game with it ([Mossfield Origins](https://store.steampowered.com/app/1836400/Mossfield_Origins/)). 

There's people using it all the time (see the discord), the nice thing about it that it has been stable and comfortable for some time, so it doesn't need constant change just for change sake. This can look like a lot of inactivity but it's more like periods of focus (on the language itself, and using it, like making games and engines). 

The plan is to collate all the longer term big work into PRs and get 0.5 out along the way, but since it's a volunteer project by people _using it_ there tends to be an ebb and flow ✨
